frontend/src/features/projects/dispose/ProjectDisposeLayout.tsx

```diff
@@ -43,11 +56,35 @@ const ProjectDisposeLayout = ({ match, location }: { match: Match; location: Loc
         }
       }
     });
+  };
 
   const getComponentPath = (wfc: ProjectWorkflowComponent) => {
     return `${match.url}${_.find(workflowStatuses, { sortOrder: wfc.workflowStatus })?.route}`;
   };
 
+  const handleSubmit = async () => {
+    const preDraftValues: any = {
+      name: formikRef.current?.values.name,
+      note: formikRef.current?.values.note,
+      description: formikRef.current?.values.description,
+      properties: [],
+      tierLevelId: 1,
+      statusId: 0,
+      agencyId: 0,
+      tasks: [],
+    };
+    if (!projectNumber) {
+      await (dispatch(createProject(preDraftValues)) as any).then((project: IProject) => {
```

Comment:
Maybe I'm missing something, but why call create project and then save project?

Reply:
Hmm yeah good point.. I'll look into it. Just commented out save project quickly to see if it would work but seemed to break something